README.md

```diff
@@ -59,7 +59,7 @@ $ swag init
 
   Make sure to import the generated `docs/docs.go` so that your specific configuration gets `init`'ed. If your General API annotations do not live in `main.go`, you can let swag know with `-g` flag.
   ```sh
-  swag init -g http/api.go
+  swag init --generalInfo=http/api.go
```

Comment:
Could you undo this change?

Reply:
@easonlin404 with latest HEAD -g seems not to work and help shows the -g acronym not:

OPTIONS:
   --generalInfo value       Go file path in which 'swagger general API Info' is written (default: "main.go")
   --dir value               Directory you want to parse (default: "./")
   --exclude value           exclude directories and files when searching, comma separated
   --propertyStrategy value  Property Naming Strategy like snakecase,camelcase,pascalcase (default: "camelcase")
   --output value            Output directory for all the generated files(swagger.json, swagger.yaml and doc.go) (default: "./docs")
   --parseVendor             Parse go files in 'vendor' folder, disabled by default (default: false)
   --parseDependency         Parse go files in outside dependency folder, disabled by default (default: false)
   --markdownFiles value     Parse folder containing markdown files to use as description, disabled by default
   --generatedTime           Generate timestamp at the top of docs.go, true by default (default: false)
   --help, -h                show help (default: false)



(Thats why I adjusted it)

Please advise

Reply:
ok I saw that a fix for this is pending - changed the line back. Thanks